Dockerfile

```diff
@@ -15,3 +15,8 @@ ENTRYPOINT poetry run flask run --host=0.0.0.0
 FROM base AS development
 ENV FLASK_DEBUG=true
 ENTRYPOINT poetry run flask run --host=0.0.0.0
+
+# CI tests
+FROM base AS tests
+ENV PYTHONPATH=/app
```

Comment:
Nice solution here!

An alternative way to solve this - that might give you some context for some existing Python-oddness - is to add an empty `__init__.py` file to the `tests` folder (and every folder that we add containing Python code). This helps Python to identify which folders are "packages" vs just folders - and Python has claimed they aren't necessary but in my experience, as here, that's not _always_ the case!

No preference for me which solution we use, but thought the context might be nice